Cython/Compiler/ExprNodes.py

```diff
@@ -31,7 +31,7 @@
 from . import Nodes
 from .Nodes import Node, utility_code_for_imports, SingleAssignmentNode
 from . import PyrexTypes
-from .PyrexTypes import py_object_type, typecast, error_type, \
+from .PyrexTypes import CTypedefType, CppClassType, py_object_type, typecast, error_type, \
```

Comment:
```suggestion
from .PyrexTypes import py_object_type, typecast, error_type, \
```

I'd propose to undo this, and just look up the names in `PyrexTypes`. The names that we deliberately import are the ones used often while these are only used once